pr_agent/servers/github.py

```diff
@@ -0,0 +1,10 @@
+##获取github数据
+class GithubData(object):
+    def __init__(self):
+        self.des = "dssdd"
+
+    def __str__(self):
+        return self.des
```

Comment:
**Suggestion:** Avoid using magic methods like '__str__' for business logic. Instead, create a separate method for it.
```suggestion
class GithubData(object):
    def __init__(self):
        self.des = "dssdd"

    def get_description(self):
        return self.des
```